src/drive/store/rootReducer.js

```diff
@@ -19,25 +19,25 @@ import {
 } from 'drive/mobile/modules/authorization/duck'
 import { default as replication } from 'drive/mobile/modules/replication/duck'
 
-const baseReducers = {
-  ui,
-  view,
-  filelist,
-  upload,
-  selection,
-  rename,
-  availableOffline
-}
-
-const mobileReducer = combineReducers({
-  authorization,
-  settings,
-  replication,
-  mediaBackup
-})
-
 // Per Dan Abramov: https://stackoverflow.com/questions/35622588/how-to-reset-the-state-of-a-redux-store/35641992#35641992
 const createRootReducer = client => {
+  const baseReducers = {
+    ui,
+    view,
+    filelist,
+    upload,
+    selection,
+    rename,
+    availableOffline
+  }
+
+  const mobileReducer = combineReducers({
+    authorization,
+    settings,
+    replication,
+    mediaBackup
+  })
+
```

Comment:
I had to move these — strangely, inside an intent, the `view` reducer was undefined at the time of import, but was perfectly fine inside the function.